scipy_doctest/impl.py

```diff
@@ -538,6 +539,19 @@ def find(self, obj, name=None, module=None, globs=None, extraglobs=None):
         # XXX: does this make similar checks in testmod/testfile duplicate?
         if module not in self.config.skiplist:   
             tests = super().find(obj, name, module, globs, extraglobs)
+
+            if inspect.isclass(obj):
+                descriptors = [
+                    (name_, method)
+                    for name_, method in inspect.getmembers(obj)
+                    if inspect.isdatadescriptor(method)
+                ]
+
+                for name_, method in descriptors:
+                    tests += super().find(
+                        method, f'{name}.{name_}', module, globs, extraglobs
+                    )
+
```

Comment:
is there a reason to collect the descriptors, or could this be rewritten as
```
for name_, method in inspect.getmembers(obj):
    if inspect.isdatadescriptor(method):
        tests += super().find(
            method, f'{name}.{name_}', module, globs, extraglobs
        )
```

Not critical, whatever you think